frontend/public/components/graphs/bar.tsx

```diff
@@ -23,13 +23,14 @@ const PADDING_RATIO = 1 / 3;
 export const BarChart: React.FC<BarChartProps> = ({
   barSpacing = 15,
   barWidth = DEFAULT_BAR_WIDTH,
-  title,
-  query,
   data = [],
+  isLink = true,
```

Comment:
Booleans that default to true tend to be confusing. I'd recommend using `noLink` and flipping the logic.

Reply:
👍 Addressed by https://github.com/openshift/console/pull/4494